playhouse/tests/models.py

```diff
@@ -336,6 +337,9 @@ class ServerDefaultModel(TestModel):
 class EmptyModel(TestModel):
     pass
 
+class PasswordTestModel(TestModel):
+    username = TextField()
+    password = PasswordField(iterations=4)
```

Comment:
This will fail if `bcrypt` is not installed. Furthermore, this should probably go in `playhouse.tests.test_fields` rather than in the "general test models" module.